office/taskwarrior-tui/Portfile

```diff
@@ -17,18 +17,30 @@ maintainers         {gmail.com:herby.gillot @herbygillot} \
                     openmaintainer
 license             MIT
 
-checksums           rmd160  f41d0ed9fd2912c5112af7000375a6d9b128ecf0 \
-                    sha256  45a173c5551fa2bc7de753e2e20272ed80c5212480e6c51c70f93c4dca519516 \
-                    size    59638
+checksums           rmd160  d0f509aca0736b6b57a73c6d72111b66c3bc2d15 \
+                    sha256  ba2222aecfc285a41440d7683370ce789bbfc9484fe9f0322f0ca36e917f82bf \
+                    size    59996
 
 build.pre_args      --release -v -j${build.jobs}
 
+post-build {
+    system -W ${worksrcpath}/docs "pandoc ${name}.1.md --standalone \
+        --to=man -o '${name}.1'"
+}
+
+depends_build-append \
+                    port:pandoc
+
 depends_run-append  port:task
 
 destroot {
     xinstall -m 755 \
         ${worksrcpath}/target/[cargo.rust_platform]/release/${name} \
         ${destroot}${prefix}/bin/
+
+    # xinstall -d 755 ${destroot}${prefix}/share/man/man1
```

Comment:
Let's delete this commented-out line if it's not needed

Reply:
Thanks, that is why I commented-out line to check that `destroot`'es create a directory by default. But then I forgot to delete this line.